setup/files/role.yml

```diff
@@ -0,0 +1,9 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: redhat-test
+  name: pod-reader
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]
```

Comment:
That role looks too restrictice. We want something that allows a developer user to deploy apps, so create and edit pods, services, etc. I thought that the OpenShift read, edit, and admin roles were standard from Kubernetes RBAC. If not, we can use the edit role as a starting point and simplify it a bit by removing references to openshift extensions.